docs/common/craft-parts/reference/plugins/dotnet_v2_plugin.rst

```diff
@@ -0,0 +1,231 @@
+.. _craft_parts_dotnet_v2_plugin:
+
+.NET plugin (v2)
+================
+
+The ``dotnet`` plugin builds .NET projects using the ``dotnet`` tool.
+
+.. note::
+    This plugin supersedes :ref:`.NET plugin <craft_parts_dotnet_plugin>`.
+
+Keywords
+--------
+
+In addition to the common :ref:`plugin <part-properties-plugin>` and
+:ref:`sources <part-properties-sources>` keywords, this plugin provides the following
+plugin-specific keywords:
+
+.. _global_flags:
+
+Global Flags
+~~~~~~~~~~~~
+
+dotnet-configuration
+^^^^^^^^^^^^^^^^^^^^
+**Type:** string
+
+**Default:** ``"Release"``
+
+The .NET build configuration to use. Possible values are ``"Debug"`` and ``"Release"``.
+
+dotnet-project
+^^^^^^^^^^^^^^
+**Type:** string
+
+**Default:** ``None``
```

Comment:
Should this be `"None"`? Or are we saying it's unset?

Reply:
We're saying it's unset, thus not present in the command if not specified. Should we say "Default: unset" instead?

Reply:
Yes please!